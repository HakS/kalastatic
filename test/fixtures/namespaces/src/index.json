{
  "title": "Hello World!",
  "description": "Greetings, Human.",
  "thebuttons": [
    {
      "title": "Button 1"
    },
    {
      "title": "Button 2"
    },
    {
      "title": "Button 3"
    }
  ]
}
