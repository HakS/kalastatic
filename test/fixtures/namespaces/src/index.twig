<!DOCTYPE html>
<html lang="en">
<head>
<title>CSS Template</title>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="main.css">
</head>
<body>
<h1>{{ title }}</h1>
<p>{{ description }}</p>

{% include "@molecules/buttongroup/buttongroup.twig" with { buttons: thebuttons } only %}

</body>
</html>
