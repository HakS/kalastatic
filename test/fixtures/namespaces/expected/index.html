<!DOCTYPE html>
<html lang="en">
<head>
<title>CSS Template</title>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="main.css">
</head>
<body>
<h1>Hello World!</h1>
<p>Greetings, Human.</p>

	<ul>
					<li>
					<li>
					<li>
			</ul>

</body>
</html>
